pom.xml

```diff
@@ -6,7 +6,7 @@
 	<artifactId>opensrp-server-api</artifactId>
 	<packaging>jar</packaging>
 
-	<version>1.1.3-SNAPSHOT</version>
+	<version>1.1.4-ALPHA-SNAPSHOT</version>
```

Comment:
remove the `ALPHA`  keyword and ensure the version is higher than that on master branch.